<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditFrame</class>
 <widget class="QDialog" name="EditFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Edit Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Brand:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Brand_3">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_18">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_model">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Color_3">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Price: $</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Price_3">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Units:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_units_3"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QCommandLinkButton" name="commandLinkButton_Update">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCommandLinkButton" name="commandLinkButton_Delete">
     <property name="maximumSize">
      <size>
       <width>179</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QCommandLinkButton" name="commandLinkButton_cancel_3">
     <property name="maximumSize">
      <size>
       <width>179</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
